test/integration/index.spec.ts

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash'
```

Comment:
Aren't `test/e2e/index.spec.ts` and this file exactly the same?

Reply:
Not exactly, `test/e2e/index.spec.ts` is loading the built package (from `dist` folder) while `test/integration/index.spec.ts` is loading the index from `src` folder